jsk_topic_tools/src/connection_based_nodelet.cpp

```diff
@@ -52,6 +60,13 @@ namespace jsk_topic_tools
     }
   }
 
+  void ConnectionBasedNodelet::warnNeverSubscribedCallback(const ros::WallTimerEvent& event)
+  {
+    if (!ever_subscribed_) {
+      NODELET_WARN("'%s' subscribes topics only with child subscribers. (`always_subscribe: true` to force it to subscribe)", nodelet::Nodelet::getName().c_str());
```

Comment:
yes. child classes need to call `onInitPostProcess` to enable `always_subscribe` feature. 
But current implementation is not good. We should remove this `onInitPostProcess`, but I have no idea.

`subscribe()` method should be called after all the publishers are instantiated.
It is difficult for parent class to know the timing.